Classes/Csv/FrontEndRegistrationAccessCheck.php

```diff
@@ -2,14 +2,16 @@
 
 declare(strict_types=1);
 
+namespace OliverKlee\Seminars\Csv;
+
```

Comment:
In this file, the leading backslash before `Tx_Oelib_FrontEndLoginManager` is still missing (according to the output of the legacy tests).